src/DynamoCore/Configuration/Configurations.cs

```diff
@@ -3,58 +3,140 @@
 
 namespace Dynamo.Configuration
 {
+    /// <summary>
+    ///     This class contains properties that are used in Dynamo.
+    /// </summary>
     public class Configurations
     {
```

Comment:
Question : Why the configurations are public? I guess these should be internal or protected. Why the API users need to know about his?